docs: add more content to "Get started" (#298)
bytebase/bytebase.com: content/docs/en/get-started/work-with-a-project/overview.md

---
title: Work with a Project
---

> After you configured in workspace, it's time to work with a **Project**.

Reviewer: ```suggestion
After you configured the workspace, it's time to work with a **Project**.
```

**Project** is the container to group logically related **Databases**, **Issues** and **Users** together. In Bytebase, A **Database** or an **Issue** always belongs to a single **Project**.

Project is similar to the project concept in other dev tools such as Jira, GitLab. 

>  Project is also the peering entity with the VCS repository to setup version control workflow.

Reviewer: ```suggestion
 Project is also the peering entity with the VCS repository to set up version control workflow.
```

> There are two roles at project level: **Owner** and **Developer**.

Reviewer: ```suggestion
There are two roles at the project level: **Owner** and **Developer**.
```

> To make changes to databases, you have to [create a project](create-a-project) first, and then [run an SQL workflow](run-a-ui-workflow).

Reviewer: ```suggestion
To make changes to databases, you have to [create a project](create-a-project) first, and then [run a SQL workflow](run-a-ui-workflow).
```
